src/Resources/skeleton/doctrine/Repository.tpl.php

```diff
@@ -1,29 +1,36 @@
-<?= "<?php\n"; ?>
+<?= "<?php" . PHP_EOL; ?>
 
-namespace <?= $namespace; ?>;
+namespace <?= $namespace ?>;
 
-use <?= $entity_full_class_name; ?>;
+use <?= $entity_full_class_name ?>;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
 use Symfony\Bridge\Doctrine\RegistryInterface;
 
 /**
+ * Class <?= $class_name . PHP_EOL; ?>
  * @method <?= $entity_class_name; ?>|null find($id, $lockMode = null, $lockVersion = null)
  * @method <?= $entity_class_name; ?>|null findOneBy(array $criteria, array $orderBy = null)
  * @method <?= $entity_class_name; ?>[]    findAll()
  * @method <?= $entity_class_name; ?>[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
  */
 class <?= $class_name; ?> extends ServiceEntityRepository
 {
+    /**
```

Comment:
We can remove this entire PHPdoc block for the same reasons given for the other blocks.